pkg/collector/corechecks/cluster/helm/helm.go

```diff
@@ -201,36 +205,6 @@ func (hc *HelmCheck) setSharedInformerFactory(apiClient *apiserver.APIClient) {
 	)
 }
 
-func (hc *HelmCheck) addExistingReleases(apiClient *apiserver.APIClient) error {
-	selector := labels.Set{"owner": "helm"}.AsSelector()
-
-	initialHelmSecrets, err := apiClient.Cl.CoreV1().Secrets(v1.NamespaceAll).List(
-		context.Background(), metav1.ListOptions{LabelSelector: selector.String()},
-	)
-	if err != nil {
-		return err
-	}
-
-	for _, secret := range initialHelmSecrets.Items {
-		hc.addSecret(&secret)
-	}
-
-	initialHelmConfigMaps, err := apiClient.Cl.CoreV1().ConfigMaps(v1.NamespaceAll).List(
-		context.Background(), metav1.ListOptions{LabelSelector: selector.String()},
-	)
-	if err != nil {
-		return err
-	}
-
-	for _, configMap := range initialHelmConfigMaps.Items {
-		hc.addConfigmap(&configMap)
-	}
-
-	hc.existingReleasesStored = true
```

Comment:
I think this breaks the datadog events emitted by this check.

This boolean is only set there, and it needs to be true to emit datadog events: 
https://github.com/DataDog/datadog-agent/pull/13013/files#diff-a91a2bb9f4306899a82675022adbdb60dcdec035844e948284139007fc4dcb83R357

Note that the purpose of this bool is to avoid emitting datadog events for Helm releases that were deployed before the agent. Otherwise, every time we start the agent will get an event like "New Helm release deployed" for all the existing releases.

I realized that maybe the comment here was not clear enough because it mentions "events", which can be confused with "kubernetes events", but I was referring to datadog events: https://github.com/DataDog/datadog-agent/pull/13013/files#diff-a91a2bb9f4306899a82675022adbdb60dcdec035844e948284139007fc4dcb83R69

Reply:
Nice catch! Added some logic to do the filtering based on the creation timestamp of the object